coalib/collecting/Collectors.py

```diff
@@ -76,7 +81,39 @@ def collect_files(file_paths, log_printer, ignored_file_paths=None,
     _warn_if_unused_glob(log_printer, file_paths, file_globs_with_files,
                          "No files matching '{}' were found.")
     limited_files = list(filter(limit_fnmatch, collected_files))
-    return limited_files
+
+    changed_files = []
+    last_coala_run = get_last_coala_run_time(log_printer, project_dir)
```

Comment:
I dislike these variable names because they have "coala" in them. Preferably just use "last_run" ?
Very subjective opinion, I know. But the word coala is unnecessary (as it's in the coala source) and just makes the variable longer.